### PR TITLE
eventub connector to support multiple kcqls

### PR DESCRIPTION
**Pull request by GoMati-MU**
#### Description

This one is to allow multi-statement KCQL support with additional features mentioned:
- check input and output topic names against regex
- disallow one-to-many mappings

#### Testing
1. Tests were changed around the changed parts of the code (e.g. Source Record mapping)
2. Additional unit tests were added

### CODE REVIEW

**Comment by stheppi** [comment]
> * disallow many-to-one mappings
 
 
I imagine you mean one-to-manny mapping :)

File: java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/streamreactor/connect/azure/eventhubs/util/KcqlConfigPortTest.java [hunk 1]
```diff
@@ -1,23 +1,73 @@
...
+      assertEquals(inputErrorMessage, e.getMessage());
+    }
+
+    try {
+      KcqlConfigPort.mapInputToOutputsFromConfig(illegalOutputKcql);
+      fail("Exception not thrown");
+    } catch (ConfigException e) {
+      assertEquals(outputErrorMessage, e.getMessage());
+    }
+
```
Comment (stheppi) [inline java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/streamreactor/connect/azure/eventhubs/util/KcqlConfigPortTest.java:71]: I think there needs to be a few more tests:

```
INSERT INTO a SELECT * FROM x;
INSERT INTO a SELECT * FROM y;
```
should be accepted


```
INSERT INTO a SELECT * FROM x;
INSERT INTO b SELECT * FROM x;
```
should not be accepted

**Review by stheppi — CHANGES_REQUESTED** [review]
See the comments and also the info on connector source `start` method where validation should happen.

File: java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/streamreactor/connect/azure/eventhubs/util/KcqlConfigPortTest.java (hunk 1 above)
Comment (GoMati-MU) [reply java-connectors/kafka-connect-azure-eventhubs/src/test/java/io/lenses/streamreactor/connect/azure/eventhubs/util/KcqlConfigPortTest.java:71]: Added those scenarios, thanks!

**Comment by GoMati-MU** [comment]
Addressed rest of the comments

**Review by stheppi — APPROVED** [review]

**Pull request merged**